ui/app/src/views/dashboard/Panel.tsx

```diff
@@ -38,6 +38,7 @@ function Panel(props: PanelProps) {
         height: '100%',
         display: 'flex',
         flexFlow: 'column nowrap',
+        overflow: 'visible',
```

Comment:
Tooltips were getting cut off before when they overlapped container boundary

Reply:
Interesting. Typically you see tooltips like this getting attached to the `body` or some other root element (i.e. MUI does this for all its popover/modals) so I wonder if we could take a similar approach so we don't let panel content overflow the container. Although that may complicate positioning... 🤔

Reply:
Yeah good point, that would also help with fixing the issue where when you move your cursor really fast, the tooltip sometimes remains visible.

It will require refactoring the assembleTransform function and how cursor coordinates are passed. I'll make a follow-up PR for this if that's alright?